frontend/packages/dev-console/src/components/topology/shapes/__tests__/__snapshots__/WorkloadNode.spec.tsx.snap

```diff
@@ -0,0 +1,1231 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TopologyUtils  should match the previous snapshot 1`] = `
```

Comment:
Is this even being used? I cannot find where the snapshot call is being made.

Reply:
Nope. Was pulling in another test. Fixed